scipy/signal/_signaltools.py

```diff
@@ -1852,9 +1852,33 @@ def medfilt2d(input, kernel_size=3):
     -----
     This is faster than `medfilt` when the input dtype is ``uint8``,
     ``float32``, or ``float64``; for other types, this falls back to
-    `medfilt`; you should use `scipy.ndimage.median_filter` instead as it is
-    much faster.  In some situations, `scipy.ndimage.median_filter` may be
+    `medfilt`. In some situations, `scipy.ndimage.median_filter` may be
     faster than this function.
+    
+    Examples
+    ------
```

Comment:
This underline should have same number of dashes as `Examples` string hence two more. The failing test tells the reason of failure.